linode/manager
# upcoming: [M3-7357] - Add AGLB Helper Text and Copy

## Description 📝

Adds copy and helper text for the AGLB project 📝

## How to test 🧪

### Prerequisites
- Turn on the MSW or test with the `dev-test-aglb` account in dev

### Verification steps 
- Check over all copy added throughout AGLB 👀
  - Check spelling, capitalization, styling  🎨
  - Things might not be 100% perfect or final, I'm doing my best to work with what we've got.

## As an Author I have considered 🤔
